test/state/mpt_hash.hpp

```diff
@@ -9,7 +9,16 @@
 namespace evmone::state
 {
 struct Account;
+struct Transaction;
+struct TransactionReceipt;
 
 /// Computes Merkle Patricia Trie root hash for the given collection of state accounts.
 hash256 mpt_hash(const std::unordered_map<address, Account>& accounts);
+
+/// Computes Merkle Patricia Trie root hash for the given collection of transaction.
+hash256 mpt_hash_txs(const std::unordered_map<uint64_t, Transaction>& transactions);
```

Comment:
Aren't the keys here always 0, 1, 2, ... ? Could just use `vector` instead then?

Reply:
Seems reasonable. Then `std::span<const Transaction>` is even better.